src/bin/firebase.js

```diff
@@ -37,27 +38,18 @@ var utils = require("../utils");
 var cmd;
 
 var logFilename = path.join(process.cwd(), "/firebase-debug.log");
-logger
-  .add(winston.transports.Console, {
-    level: process.env.DEBUG ? "debug" : "info",
-    showLevel: false,
-    colorize: true,
-  })
-  .add(winston.transports.File, {
+process.env.DEBUG = _.includes(args, "--debug") || process.env.DEBUG;
```

Comment:
Normally we would check `options.debug` instead of inspecting `args` ... do you have access to `options` here?

Reply:
this is just a copy / paste of how we were checking before, I think this might be before `options` exists since it's not in a command, but I'll look